packages/theme-chalk/src/select.scss

```diff
@@ -159,6 +159,7 @@
 
   @include e(placeholder) {
     position: absolute;
+    z-index: -1;
```

Comment:
What is the purpose of setting `z-index` to `-1` here? 🤔

Reply:
If the placeholder is on top of the native input, it will affect the right-click menu content of the mouse

Reply:
Okay.